FBC/FunctionalEntities/NorthAmericanEntities/USRegulatoryAgencies.rdf

```diff
@@ -318,15 +318,27 @@
 	<owl:NamedIndividual rdf:about="&fibo-fbc-fct-usjrga;AmericanBankersAssociationRegistrationAuthority">
 		<rdf:type rdf:resource="&fibo-fbc-fct-ra;RegistrationAuthority"/>
 		<rdf:type rdf:resource="&fibo-fbc-pas-fpas;FinancialServiceProvider"/>
-		<rdfs:label>American Bankers Association (ABA) Registration Authority (RA)</rdfs:label>
+		<rdfs:label>American Bankers Association Registration Authority</rdfs:label>
 		<rdfs:isDefinedBy rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/FunctionalEntities/NorthAmericanEntities/USRegulatoryAgencies/"/>
-		<skos:definition>American Bankers Association (ABA) Registration Authority, which is a function of the ABA for registration of issuer identification numbers in the US</skos:definition>
+		<skos:definition>American Bankers Association (ABA) Registration Authority (RA), which is a function of the ABA for registration of issuer identification numbers in the US</skos:definition>
 		<fibo-fnd-rel-rel:hasIdentity rdf:resource="&fibo-fbc-fct-usjrga;AmericanBankersAssociation"/>
 		<fibo-fnd-rel-rel:manages rdf:resource="&fibo-fbc-fct-usjrga;ABAIINRegistry"/>
 		<fibo-fnd-utl-av:abbreviation>ABA RA</fibo-fnd-utl-av:abbreviation>
 		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">http://www.investopedia.com/terms/i/issuer-identification-number-iin.asp</fibo-fnd-utl-av:adaptedFrom>
 	</owl:NamedIndividual>
 	
+	<owl:NamedIndividual rdf:about="&fibo-fbc-fct-usjrga;BoardOfGovernorsOfTheFederalReserveSystem">
+		<rdf:type rdf:resource="&fibo-be-oac-exec;BoardOfDirectors"/>
+		<rdf:type rdf:resource="&fibo-fbc-fct-ra;RegistrationAuthority"/>
+		<rdf:type rdf:resource="&fibo-fbc-fct-rga;RegulatoryAgency"/>
+		<rdfs:label>Board of Governors of the Federal Reserve System</rdfs:label>
+		<rdfs:isDefinedBy rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/FunctionalEntities/NorthAmericanEntities/USRegulatoryAgencies/"/>
+		<skos:definition>regulatory agency and registration authority role of the Federal Reserve System</skos:definition>
+		<fibo-fnd-rel-rel:hasIdentity rdf:resource="&fibo-fbc-fct-usjrga;FederalReserveBoard"/>
```

Comment:
The Board of Governors is a distinct agency, not a role. See https://www.federalreserve.gov/faqs/about_12593.htm Maybe it's the name that is the issue and FederalReserveBoardAsRegulator would make more sense.

If the role is so important then don't we need a separate Role to be the object of isProducedBy triples for interest rates?

In either case it should not be typed BoardOfDirectors since that has the definition "Governing body (called the board) of an incorporated firm." and the rest of its skos:definition is all about corporations and doesn't apply either.

Reply:
There are two individuals - the independent party one is called FederalReserveBoard, fyi, so this one is the party in role, but I'll get rid of the BoD and fix the definition.

Reply:
OK but what about my 2nd para - which was whether we need a separate PartyInRole for the FRB as producer of  interest rates?

Reply:
I think no, because they are a central bank.  If that were not the case then I would agree with you.  And, if we need that, I would add it in IND (which maybe I did - will see when we merge this with what's in IND, where I know there will be conflicts to correct among other things).

Reply:
In other words, I would do this in IND, and should do so for all of the international / central banks that we have individuals for, not just the Fed.  We have a couple of others - Bank of England and I think the European Central Bank, but will confirm that and add the role in IND for each of the ones we have.  Also Bank of Canada.